ChargingStation.Backend/Infrastructure/ChargingStation.Infrastructure/AzureTableStorage/BaseTableStorageEntity.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Azure;
+using Azure.Data.Tables;
+
+namespace ChargingStation.Infrastructure.AzureTableStorage
+{
+    public class BaseTableStorageEntity : ITableEntity
+    {
+        public string PartitionKey { get; set; }
+        public string RowKey { get; set; }
+        public DateTimeOffset? Timestamp { get; set; }
+        public ETag ETag { get; set; }
+
+        BaseTableStorageEntity(string partitionKey, string rowKey, DateTimeOffset? timestamp, ETag eTag)
+        {
+            PartitionKey = partitionKey;
+            RowKey = rowKey;
```

Comment:
You can use LogTail Pattern for row keys
https://learn.microsoft.com/en-us/azure/storage/tables/table-storage-design-patterns#log-tail-pattern